README.md

```diff
@@ -2,38 +2,26 @@
 
 **Maintainer**: [@PiotrBerebecki](https://github.com/PiotrBerebecki)
 
-# Express Handlebars.js code-along workshop
+# Express Handlebars.js Workshop
 
 This is a workshop designed as a code along to learn about html templating using Handlebars.js.
```

Comment:
Can we remove reference to 'code along' in line 7 too?

Reply:
nice catch, done